colcon_devtools/verb/version_check.py

```diff
@@ -2,7 +2,7 @@
 # Licensed under the Apache License, Version 2.0
 
 import sys
-from xmlrpc.client import ServerProxy
+import requests
```

Comment:
Why does this need to use an external package? `urllib.request.urlopen()` and `json.load()` seem to be sufficient on a first look.

Reply:
Very true, I just defaulted to `requests` since it was familiar, changed to built-ins: 778a263